shotover/src/transforms/query_counter.rs

```diff
@@ -16,6 +18,7 @@ use super::UpChainProtocol;
 #[derive(Clone)]
 pub struct QueryCounter {
     counter_name: &'static str,
+    counters: HashMap<&'static str, Counter>,
```

Comment:
HashMap should be a `HashMap<String, Counter>`.
I previously said it should be `HashMap<&'static str, Counter>` but that was incorrect.
I saw that `.short_name()` returned an &'static str for the cassandra logic.
But I missed that `redis_query_name` returned a String for the redis logic.

Using `String` instead of &'static str will increase the cost of inserting an element, but it will not affect the cost of retrieving an element which is where we are most interested.